ui/app/AppLayouts/Communities/panels/SharedAddressesAccountSelector.qml

```diff
@@ -19,7 +19,13 @@ StatusListView {
 
```

Comment:
I think this component is by far the heaviest one in therms of model operations 😄 
We'd better not instantiate it unless it's requested by the user.

Reply:
Yeah, it's wrapped in a `Component` inside `SharedAddressesPanel`